src/Spryker/Zed/EventBehavior/Business/Model/TriggerManagerInterface.php

```diff
@@ -7,12 +7,14 @@
 
 namespace Spryker\Zed\EventBehavior\Business\Model;
 
+use Generated\Shared\Transfer\EventTriggerResponseTransfer;
+
 interface TriggerManagerInterface
 {
     /**
-     * @return void
+     * @return \Generated\Shared\Transfer\EventTriggerResponseTransfer
      */
-    public function triggerRuntimeEvents();
+    public function triggerRuntimeEvents(): EventTriggerResponseTransfer;
```

Comment:
This needs reverting then?

Reply:
No, this is not part of the API. The API uses this class and the old method will not return only the new one will return this response.